lib/index.js

```diff
@@ -140,7 +142,8 @@ Keen.prototype.identify = function(identify) {
   var props = { user: user };
   this.addons(props, identify);
   this.client.setGlobalProperties(function() {
-    return props;
+    // Clone the props so the Keen Client can't manipulate the ref
+    return JSON.parse(JSON.stringify(props));
```

Comment:
@tsholmes yeah agreed with this. @josephwegner our apologies for responding so late on this PR!